src/db.js

```diff
@@ -498,12 +498,10 @@
             return;
         }
 
-        for (var objectStoreKey in db.objectStoreNames) {
-            if (db.objectStoreNames.hasOwnProperty(objectStoreKey)) {
-                var name = db.objectStoreNames[objectStoreKey];
-                if (schema.hasOwnProperty(name) === false) {
-                    e.currentTarget.transaction.db.deleteObjectStore(name);
-                }
+        for (var i = 0; i < db.objectStoreNames.length; i++) {
+            var name = db.objectStoreNames[i];
+            if (schema.hasOwnProperty(name) === false) {
```

Comment:
Was someone else's PR... Thought I fixed that already, but maybe it was in one of the branches I am working on.